### PR TITLE
WW-161: Set appreciate response format as json

### PR DESCRIPTION
**Pull request by lukaszkonieczny**
To avoid rendering response html from non-existing template

@Wikia/west-wing

### CODE REVIEW

File: extensions/wikia/ContributionAppreciation/ContributionAppreciationController.class.php [hunk 1]
```diff
@@ -19,6 +19,8 @@ public function appreciate() {
 
 			$this->sendMail( $revisionId );
 		}
+
+		$this->response->setFormat( WikiaResponse::FORMAT_JSON );
```
Comment (sebastianmarzjan) [inline extensions/wikia/ContributionAppreciation/ContributionAppreciationController.class.php:28]: And what is the response body?

**Comment by sebastianmarzjan** [comment]
👍

**Comment by rybmat** [comment]
LGTM

**Comment by rybmat** [comment]
Tests failures not connected with this PR

**Pull request merged**